samples/nrf9160/nrf_cloud_rest_agps/README.rst

```diff
@@ -0,0 +1,350 @@
+   :local:
+   :depth: 2
+
+The REST A-GPS sample demonstrates how the `nRF Cloud`_ Assisted GPS (`A-GPS`_) feature can be used to implement A-GPS in your application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample also requires an nRF Cloud account.
+
+.. include:: /includes/spm.txt
+
+.. note::
+   This sample does not support the modem firmware v1.1.x (or earlier).
+
+Overview
+********
+
+The use of A-GPS speeds up the time to first GPS position fix (commonly known as TTFF or Time to first fix), as the required satellite information is downloaded from the A-GPS source instead of direct download of the data from the satellites.
+
+The Assistance GPS data includes the following information:
+
+* `Ephemeris`_
+* `Almanac`_
+* UTC time
+* Integrity data
+* Approximate location
+* GPS system clock
+* Ionospheric correction parameters for `Klobuchar Ionospheric Model`_
+
+.. note::
+
+   .. include:: ../../../doc/nrf/libraries/networking/nrf_cloud_agps.rst
+      :start-after: agpslimitation_start
+      :end-before: agpslimitation_end
+
+TTFF is expected to be less when compared to that in the scenario of GPS without assistance data, performing a cold or a warm start.
+The TTFF is expected to be comparable to that of a GPS hot start and is usually in the range of 5 to 20 seconds in good satellite signaling conditions.
+Good satellite signaling conditions in this context means that the device is positioned in a way to have a direct line of sight to a large portion of the sky.
+
+Note that the radio component on the nRF9160-based device is shared between LTE and GPS and hence only one of these can be active at any moment of time.
+LTE has priority over the radio resources, and GPS operates in the time interval between LTE required operations as per LTE specification.
+Consequently, GPS is blocked from searching as long as the modem is in `RRC connected mode <RRC idle mode_>`_.
+
+The modem is in RRC connected mode when sending and receiving data over the LTE network.
+After data transmission is complete, the modem stays in RRC connected mode for a period of network-specific inactivity time, before entering RRC idle mode.
+The inactivity time varies between networks.
+It is typically in the range of 5 to 70 seconds for a specific network.
+In RRC idle mode, the GPS is usually able to operate.
+
+It is recommended to use LTE Power Saving Mode (PSM) and extended Discontinuous Reception (eDRX) mode to increase the allowed time of operation for GPS.
+During the defined intervals of PSM and eDRX, LTE communication does not occur, and the GPS has full access to the radio resources.
+In this sample, both PSM and eDRX are enabled by default.
+You can enable or disable these features by using the :kconfig:`CONFIG_LTE_POWER_SAVING_MODE` and :kconfig:`CONFIG_LTE_EDRX_REQ` configuration options.
```

Comment:
I'm confused by your suggestion.  I do not find any other example anywhere in the sdk-nrf repository where :option: is used in an .rst file.

Reply:
This is a fairly new thing to myself as well. As this option is not introduced in Kconfig, we need to refer to the definition given in the same document a bit below. @umapraseeda, please confirm I have understood this correctly.

Reply:
So, this option is not included in the Kconfig file at all. If you decide to add it, this can remain as it was :kconfig:, but due to some recent changes in the doc framework, you would need to change the line 90. I'll make a suggestion accordingly on that line.

Reply:
I copied the readme from another sample and edited it.  I hadn't noticed that the CONFIG_LTE_POWER_SAVING_MODE option was missing from my sample.